### ISSUE TITLE
video call attended before ringing

### ISSUE THREAD

**Issue by bp2468** [InnovateAsterisk/Browser-Phone#571]
when User A makes a video call to user B, even before it becomes ringing on user B, user A is in attended state and timer starts running.
![image](https://github.com/user-attachments/assets/5add322b-2f16-473a-8bec-a9775fec9821)

**Comment by InnovateAsterisk** [comment]
This may be from the SIP messages received. The system will respond to the SIP messages like 180, 183, etc, to indicate to the user the progress of the call. When the system receives a 200 OK, it will consider the call in progress and display the Video screen.

**Comment by bp2468** [comment]
This issue is resolved by removing Answer() application from the dialplan.

**Comment by InnovateAsterisk** [comment]
Yes, you cannot issue Answer() in the dial plan, this is for the agent to answer the call.

Answer is used it you want the Asterisk system to answer the call, and send media to the caller, like an IVR or to put the caller in a queue.
